spec/models/provider_enrichment_spec.rb

```diff
@@ -22,14 +22,41 @@
   end
 
   describe 'auditing' do
-    it { should be_audited }
+    it { should be_audited.except(:json_data) }
 
     it 'can link to Provider' do
       provider = create(:provider)
       provider_enrichment = create(:provider_enrichment, provider: provider)
 
       expect(provider_enrichment.audits.last.associated).to eq(provider)
     end
+
+    context 'when dealing with a `jsonb_accessor` attribute' do
+      let(:accrediting_provider_enrichment) do
+        {
+          "errors" => {}, "UcasProviderCode" => "ABC", "Description" => "A test value",
+          "validation_context" => nil
+        }
+      end
+      let(:provider_enrichment) { create(:provider_enrichment) }
+
+      before do
+        provider_enrichment.accrediting_provider_enrichments = [accrediting_provider_enrichment]
+        provider_enrichment.save
+      end
+
+      it 'does not audit `json_data` attribute' do
+        expect(provider_enrichment.audits.last.audited_changes).to_not have_key('json_data')
+      end
+
+      it 'does include `accrediting_provider_enrichments` at top level of audited_changes' do
+        changes = provider_enrichment.audits.last.audited_changes
+        expect(changes['accrediting_provider_enrichments']).to_not eq([
+          nil,
+          accrediting_provider_enrichment
+        ])
```

Comment:
Would it be worth making this less brittle?

```
        expect(changes['accrediting_provider_enrichments']).to_not include([
           nil,
           accrediting_provider_enrichment
         ])
```

Reply:
Wait, I'm missing something here. The spec says "_does_ include `accrediting_provider_enrichments` ...", how is the `to_not eq([...])` testing that?

Reply:
Good catch - sorry about that.

Reply:
Harking back to another discussion about `let`, and your comment about `its`, that's one nice thing about using an `its` here (but not saying we should change it now):

```
      its(['accrediting_provider_enrichments']) { should include(nil,[accrediting_provider_enrichment]) }
```

There would be no chance for a miss-match here. :)